app/models/User.java

```diff
@@ -0,0 +1,91 @@
+package models;
+
+/**
+ * Created by Ejub on 31.1.2016.
+ */
+import play.data.format.Formats;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "abh_user")
+public class User extends com.avaje.ebean.Model {
+    @Id
+    public String email;
+    public String lastName;
+    public String firstName;
+    public String password;
+    @OneToOne()
+    @JoinColumn(name="email")
+    public Token authToken;
+    public boolean isConfirmed;
+    public int phone;
+    public String gender;
+    @Formats.DateTime(pattern="dd/MM/yyyy")
+    public Date birthdate;
+    @OneToOne()
+    @JoinColumn(name="email")
+    public Address address;
```

Comment:
Never use public access modifiers for class fields. Make them accessible only through getters and setters. This is OOP best practice and standard!